### PR TITLE
8265612: revise the help info for jmap histo command

### PR DESCRIPTION
**Pull request by linzang**
This PR  revise the help message of the `parallel=[N]` option of command `jmap -histo`. It mainly comes from the discussion at  https://github.com/openjdk/jdk/pull/2379#issuecomment-782609582  and https://github.com/openjdk/jdk/pull/2261#issuecomment-823623186.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8265612](https://bugs.openjdk.java.net/browse/JDK-8265612): revise the help info for jmap histo command

### Reviewers
 * [Chris Plummer](https://openjdk.java.net/census#cjplummer) (@plummercj - **Reviewer**)
 * [Serguei Spitsyn](https://openjdk.java.net/census#sspitsyn) (@sspitsyn - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3598/head:pull/3598` \
`$ git checkout pull/3598`

Update a local copy of the PR: \
`$ git checkout pull/3598` \
`$ git pull https://git.openjdk.java.net/jdk pull/3598/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3598`

View PR using the GUI difftool: \
`$ git pr show -t 3598`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3598.diff">https://git.openjdk.java.net/jdk/pull/3598.diff</a>

</details>

### CODE REVIEW

**Review by plummercj — APPROVED** [review]
The changes look good, but in the review for PR 2261 I suggested you also move some other minor edits in JMap.java from that PR to this one:

https://github.com/openjdk/jdk/pull/2261/files#r617063125

File: src/jdk.jcmd/share/classes/sun/tools/jmap/JMap.java [hunk 1]
```diff
@@ -228,8 +228,8 @@ private static void dump(String pid, String options)
             } else if (subopt.startsWith("gz=")) {
                compress_level = subopt.substring("gz=".length());
                if (compress_level.length() == 0) {
-                    System.err.println("Fail: no number provided in option: '" + subopt + "'");
-                    usage(1);
+                   System.err.println("Fail: no number provided in option: '" + subopt + "'");
+                   usage(1);
```
Comment (plummercj) [inline src/jdk.jcmd/share/classes/sun/tools/jmap/JMap.java:232]: Actually lines 231 and 232 have the correct indentation. It's 229, 230, and 233 that are wrong. They need to be indented one more space.

**Review by plummercj — APPROVED** [review]

**Review by sspitsyn — APPROVED** [review]
Hi Lin,
It looks good to me. Sorry for being late on this review.
Thanks,
serguei

**Comment by linzang** [comment]
Dear @sspitsyn @plummercj 
Thanks a lot for your help reviewing. May I ask your help to push it?

BRs,
Lin

**Comment by sspitsyn** [comment]
Hi Lin,
I've added a comment with the /sponsor, so you can push it now.

**Comment by linzang** [comment]
Hi @sspitsyn，Thanks a lot！

**Comment by linzang** [comment]
Hi Serguei, 
Sorry that I forgot to add /integrate tag before asking your help sponsor, it seems I need your help to mark this PR again with /sponsor tag, Would you like to help again. Thanks a lot!
Lin

**Pull request closed without merging**